plugins/org.eclipse.elk.alg.radial/src/org/eclipse/elk/alg/radial/intermediate/rotation/AngleRotation.java

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Kiel University and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 
+ *******************************************************************************/
+package org.eclipse.elk.alg.radial.intermediate.rotation;
+
+import org.eclipse.elk.alg.radial.InternalProperties;
+import org.eclipse.elk.alg.radial.options.RadialOptions;
+import org.eclipse.elk.core.math.KVector;
+import org.eclipse.elk.graph.ElkNode;
+
+/**
+ * Rotates the entire layout around the origin to a set target angle.
+ *
+ */
+public class AngleRotation implements IRadialRotator {
+
+    /* (non-Javadoc)
+     * @see org.eclipse.elk.alg.radial.intermediate.rotation.IRadialRotator#rotate(org.eclipse.elk.graph.ElkNode)
+     */
+    @Override
+    public void rotate(ElkNode graph) {
+        double targetAngle = graph.getProperty(RadialOptions.TARGET_ANGLE);
+        
+        if (graph.getProperty(RadialOptions.COMPUTE_ADDITIONAL_WEDGE_SPACE)) {
+            // additionally we want to get half the angle of the first wedge to leave space
```

Comment:
Is this one sentence? Or are these two? Please use punctuation for such comments.